server/sonar-web/src/main/js/apps/issues2/components/App.js

```diff
@@ -118,6 +121,95 @@ export default class App extends React.PureComponent {
     this.mounted = false;
   }
 
+  attachShortcuts() {
+    window.addEventListener('keydown', this.handleKeyDown, false);
+  }
+
+  detachShortcuts() {
+    window.removeEventListener('keydown', this.handleKeyDown);
+  }
+
+  handleKeyDown = (event: KeyboardEvent) => {
+    switch (event.keyCode) {
+      case 38:
+        // up
+        event.preventDefault();
+        this.selectPreviousIssue();
+        break;
+      case 40:
+        // down
+        event.preventDefault();
+        this.selectNextIssue();
+        break;
+      case 39:
+        // right
+        event.preventDefault();
+        this.openSelectedIssue();
+        break;
+      case 37:
+        // left
+        event.preventDefault();
+        this.closeIssue();
+        break;
+    }
+  };
+
+  getSelectedIndex(): ?number {
+    const { issues, selected } = this.state;
+    return issues != null && selected != null && issues.includes(selected)
+      ? issues.indexOf(selected)
```

Comment:
I think it would be better to use only `indexOf` here and if the result is `-1` you return `null`. Instead of using both `includes` and `indexOf`